### ISSUE TITLE
An integrative ENCODE resource for cancer genomics

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#6]
## TL;DR
This paper develops custom annotation leveraging advanced assays, such as eCLIP, Hi-C, and whole-genome STARR-seq on a number of data-rich ENCODE cell types. A key aspect of this annotation is comprehensive and experimentally derived networks of both transcription factors and RNA-binding proteins (TFs and RBPs).

### Paper Link

https://www.nature.com/articles/s41467-020-14743-w

### Author/Institution

Jing Zhang (Yale University)

## Overview

The 2012 ENCODE release provided comprehensive functional genomics data, such as RNA-seq, histone modification and transcription factor (TF) ChIP-seq, and DNase-seq, to annotate the noncoding regions in the human genome. After the release, the cancer genomics community embraced the ENCODE data, together with other functional genomic data, to study the mutational landscape and regulatory networks in cancer. The current ENCODE release provides a data-rich context for a subset of cell types. The paper focuses on these data-rich cell types, we developed an integrative and network-associated annotation, which may serve as a valuable resource for cancer genomics.

## Cite

Zhang, J., Lee, D., Dhiman, V. et al. An integrative ENCODE resource for cancer genomics. Nat Commun 11, 3696 (2020). https://doi.org/10.1038/s41467-020-14743-w
